app/src/main/java/ch/heigvd/pro/b04/android/network/RockinAPI.java

```diff
@@ -1,13 +1,24 @@
 package ch.heigvd.pro.b04.android.network;
 
+import ch.heigvd.pro.b04.android.datamodel.Poll;
+import ch.heigvd.pro.b04.android.datamodel.Session;
 import ch.heigvd.pro.b04.android.datamodel.SessionCode;
 import ch.heigvd.pro.b04.android.datamodel.Token;
 import retrofit2.Call;
 import retrofit2.http.Body;
+import retrofit2.http.GET;
 import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
 
 public interface RockinAPI {
 
     @POST("/connect")
     Call<Token> postConnect(@Body SessionCode code);
+
+    @GET("/session")
+    Call<Session> getSession(@Query("token") String userToken);
+
+    @GET("/mod/{mod}/poll/{poll}")
```

Comment:
Can we keep the same format as in https://github.com/heig-PRO-b04/java-backend/wiki/APIv1.Objects ?